checks/evaluation/branch_protection.go

```diff
@@ -30,6 +30,8 @@ const (
 	nonAdminContextLevel        = 2 // Level 3.
 	nonAdminThoroughReviewLevel = 1 // Level 4.
 	adminThoroughReviewLevel    = 1 // Level 5.
+	codeownerReviewLevel        = 1 // Level 5.
```

Comment:
this should be a new level if you want a new level. The sum of all the levels should be 10. 

How bout packing in an existing level? 
One issue with adding a level 6 is that it implies a dependence on the previous levels,  meaning it will only be satisfied if all the previous levels are satisfied. 
In practice ,it's hard for us to argue that these levels make sense... we tried to adjust them in a way that sort of does, but it's hard :)

That's why I'm trying to find a place within a previous level to do it. I think we could put it as part of level 4. Level 5 is virtually non-attainable unless you enable force LGTM after each commit in a single PR (which is un-usable)

Reply:
SGTM.